lib/matplotlib/font_manager.py

```diff
@@ -266,8 +266,11 @@ def _get_fontconfig_fonts():
 @lru_cache
 def _get_macos_fonts():
     """Cache and list the font paths known to ``system_profiler SPFontsDataType``."""
-    d, = plistlib.loads(
-        subprocess.check_output(["system_profiler", "-xml", "SPFontsDataType"]))
+    try:
```

Comment:
Was this change supposed to be included?

Reply:
Apologies, that slipped in when squasing. Fixed now